API.md

```diff
@@ -18,9 +18,9 @@ Used to declare models, knex instances, and migration directory information on a
 
   - An objection or [schwifty model class](#schwiftymodel), or an array of such model classes associated with the current plugin or root server.
   - An object specifying,
-    - `knex` - a knex instance or [configuration](http://knexjs.org/#Installation-client) associated with the current plugin or root server.  It may only be specified once per plugin/server.
+    - `knex` - a knex instance or [configuration](https://knexjs.org/#Installation-client) associated with the current plugin or root server.  It may only be specified once per plugin/server.
     - `models` - An array of objection or [schwifty model classes](#schwiftymodel) associated with the current plugin or root server.
-    - `migrationsDir` - specifies a directory of knex migrations associated with the current plugin or root server.  The directory path may be either absolute, relative to the plugin's [path prefix](https://github.com/hapijs/hapi/blob/master/API.md#server.path()) when set, or otherwise relative to the current working directory.  It cannot be specified more than once within a plugin or on the root server.
+    - `migrationsDir` - specifies a directory of knex migrations associated with the current plugin or root server.  The directory path may be either absolute, relative to the plugin's [path prefix](https://hapi.dev/api/#-serverpathrelativeto) when set, or otherwise relative to the current working directory.  It cannot be specified more than once within a plugin or on the root server.
```

Comment:
```suggestion
    - `migrationsDir` - specifies a directory of knex migrations associated with the current plugin or root server.  The directory path may be either absolute, relative to the plugin's [path prefix](https://hapi.dev/api/#server.path()) when set, or otherwise relative to the current working directory.  It cannot be specified more than once within a plugin or on the root server.
```